Blockzilla/SettingsViewController.swift

```diff
@@ -431,10 +461,10 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     }
 
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        var groupingOffset = 16
+        var groupingOffset = 3
         
-        if sections[section] == .search || sections[section] == .mozilla {
-            groupingOffset = 3
+        if sections[section] == .privacy {
+            groupingOffset = 16
```

Comment:
nit: I know this one wasn't you, but we should make magic numbers constants as we update them :)